src/ScopedCommand.js

```diff
@@ -0,0 +1,26 @@
+import Command from "./Command";
+import minimatch from "minimatch";
+
+export default class ScopedCommand extends Command {
```

Comment:
I think it is the perfect problem to be solved by inheritance. Especially as then the concerns are also clearly separated for the tests as well, but I am happy to change it to whatever form you like it in order to get it integrated. Would you like me to change it to a helper?